### ISSUE TITLE
Menuhax suddenly disabled: cannot access homebrew with menuhax

### ISSUE THREAD

**Issue by AterLuna** [yellows8/3ds_homemenuhax#225]
My firmware version is 11.0.0-29K, and I installed homebrew and menuhax 4 months ago.
It worked well for 4 months, but suddenly disabled within 30 minutes.
I tried to edit save data of Pokemon: moon, so I extracted save data and edited it with PKHeX.
After editing save data, I tried to import it, however it became unable to access homebrew with menuhax.
The only suspicious point is that I turned on 3ds once without sd card, however I didn't updated 3ds, or changed the files in sd card.
Could someone help me?

**Comment by yellows8** [comment]
Only option is to use other haxx.

**Comment by ioncorimenia** [comment]
Yeah, had the same thing happen to me randomly. I was able to reinstall via browserhax, though. I have A9LH now, so I can't supply more info. I suspect that theme data got corrupted by a non-graceful unmount. Did you ever have a message that says not to eject your SD card while the system is running, or your SD card games didn't show up?
